test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -110,6 +110,19 @@ class HttpStream : public LinkedObject<HttpStream> {
   // explore these violations via MutateAction and SwapAction at the connection
   // buffer level.
   enum class StreamState : int { PendingHeaders, PendingDataOrTrailers, Closed };
+  static absl::string_view streamStateToString(StreamState state) {
```

Comment:
nit: if this was non-test code I would suggest using an array/vector of strings and return the `enumToInt`.